internal/kafka/command_topic.go

```diff
@@ -142,6 +145,40 @@ func (c *command) provisioningClusterCheck(lkc string) error {
 	return nil
 }
 
+func (c *command) prepareAnonymousContext(cmd *cobra.Command) error {
+	kafkaBootstrap, err := cmd.Flags().GetString("kafka-bootstrap")
+	if err != nil {
+		return err
+	}
+
+	platform := &config.Platform{
+		Server: kafkaBootstrap,
+		Name:   strings.TrimPrefix(kafkaBootstrap, "https://"),
+	}
+
+	kafkaClusterCfg := &config.KafkaClusterConfig{
+		ID:        "anonymous-id",
+		Name:      "anonymous-cluster",
```

Comment:
Why do these need to be set? Is there any way we can avoid that?

Reply:
i followed the standard of creating a context for anonymous cluster. May wanna refactor them together when we talk about removing `context create`

Reply:
My worry is that when we remove `context create`, this code will still exist and will prevent us from being able to easily refactor it

Reply:
we can remove `name`, but `id` is still required. i can write a comment here.

Reply:
Any way we can get around `id` and other fields being required?

Reply:
I don't think so. It'll being referenced as the key in the map